test/unit/sthE2E_test.js

```diff
@@ -0,0 +1,142 @@
+                    sth.sthServer.startServer(sthConfig.STH_HOST, sthConfig.STH_PORT, function() {
+                        sthDatabase.closeConnection(function(err) {
+                            if (err) {
+                                return done(err);
+                            }
+                            contextResponseNumericWithFixedTimeInstantV1 = require('./contextResponses/V1contextResponseNumericWithFixedTimeInstant');
+                            done();
+                        });
+                    });
+                }
+            );
+        });
+
+        after(function(done) {
+            sth.sthServer.stopServer(function() {
+                done();
+            });
+        });
+
+        it('should return 500 when GET http request (e2e)', function(done) {
+            request(
+                {
+                    uri: sthTestUtils.getURL(
+                        sthTestConfig.API_OPERATION.READ,
+                        {
+                            lastN: 0,
+                            dateFrom:
+                                contextResponseNumericWithFixedTimeInstantV1.contextResponses[0].contextElement
+                                    .attributes[0].metadatas[0].value,
+                            dateTo:
+                                contextResponseNumericWithFixedTimeInstantV1.contextResponses[0].contextElement
+                                    .attributes[0].metadatas[0].value
+                        },
+                        contextResponseNumericWithFixedTimeInstantV1.contextResponses[0].contextElement.attributes[0]
+                            .name
+                    ),
+                    method: 'GET',
+                    headers: {
+                        Accept: 'application/json',
+                        'Content-Type': 'application/json',
+                        'Fiware-Service': sthConfig.DEFAULT_SERVICE,
+                        'Fiware-ServicePath': sthConfig.DEFAULT_SERVICE_PATH
+                    }
+                },
+                function(err, response) {
+                    expect(err).to.equal(null);
+                    expect(response.statusCode).to.equal(500);
+                    expect(JSON.parse(response.body).message).to.equal('DB not connected');
+                    done(err);
+                }
+            );
+        });
+
+        it('should return 500 when DELETE http request (e2e)', function(done) {
+            request(
+                {
+                    uri: sthTestUtils.getURL(
+                        sthTestConfig.API_OPERATION.DELETE,
+                        {
+                            entityType: 'T',
+                            entityId: 'E',
+                            attrName: 'A'
+                        },
+                        'A' // maybe unneeded in API_OPERATION.DELETE case (see getURL method)
```

Comment:
I already checked that, without this line, tests complete successfully 
```suggestion

```

Reply:
The `,` in L122 should be removed if L123 is removed, shouldn't be?

Reply:
Yes, was a typo on the review

Reply:
Fixed in https://github.com/telefonicaid/fiware-sth-comet/pull/583/commits/8c49723e911bedd891c322bbe2abb7c07645b7f8